src/manageiq_client/api.py

```diff
@@ -622,6 +622,15 @@ def _process_result(self, result):
             if "id" in result:
                 d["href"] = "{}/{}".format(collection._href, result["id"])
             return Entity(collection, d)
+        elif "task_href" in result or "task_id" in result:
+            collection = getattr(self.api.collections, "tasks")
```

Comment:
@mkoura why not just `self.api.collections.tasks`? I know that above we're using `getattr` but I am wondering why we keep bringing this syntax forward.